src/components/manage/Blocks/Video/VideoSidebar.jsx

```diff
@@ -1,218 +1,51 @@
-              </>
-            )}
-            <Form.Field inline required={required}>
-              <Grid>
-                <Grid.Row>
-                  <Grid.Column width="4">
-                    <div className="wrapper">
-                      <label htmlFor="field-align">
-                        <FormattedMessage
-                          id="Alignment"
-                          defaultMessage="Alignment"
-                        />
-                      </label>
-                    </div>
-                  </Grid.Column>
-                  <Grid.Column width="8" className="align-tools">
-                    <AlignBlock
-                      align={data.align}
-                      onChangeBlock={onChangeBlock}
-                      data={data}
-                      block={block}
-                    />
-                  </Grid.Column>
-                </Grid.Row>
-              </Grid>
-            </Form.Field>
-          </Segment>
-          {data.url.match('.mp4') && (
-            <Accordion fluid styled className="form">
-              <Accordion.Title
-                active={activeAccIndex === 0}
-                index={0}
-                onClick={handleAccClick}
-              >
-                Link Settings
-                {activeAccIndex === 0 ? (
-                  <Icon name={upSVG} size="20px" />
-                ) : (
-                  <Icon name={downSVG} size="20px" />
-                )}
-              </Accordion.Title>
-              <Accordion.Content active={activeAccIndex === 0}>
-                <TextWidget
-                  id="link"
-                  title={intl.formatMessage(messages.LinkTo)}
-                  required={false}
-                  value={flattenToAppURL(data.href)}
-                  icon={data.href ? clearSVG : navTreeSVG}
-                  iconAction={
-                    data.href
-                      ? () => {
-                          onChangeBlock(block, {
-                            ...data,
-                            href: '',
-                          });
-                        }
-                      : () => openObjectBrowser({ mode: 'link' })
-                  }
-                  onChange={(name, value) => {
-                    onChangeBlock(block, {
-                      ...data,
-                      href: flattenToAppURL(value),
-                    });
-                  }}
-                />
-                <CheckboxWidget
-                  id="openLinkInNewTab"
-                  title={intl.formatMessage(messages.openLinkInNewTab)}
-                  value={data.openLinkInNewTab ? data.openLinkInNewTab : false}
-                  onChange={(name, value) => {
-                    onChangeBlock(block, {
-                      ...data,
-                      openLinkInNewTab: value,
-                    });
-                  }}
-                />
-              </Accordion.Content>
-            </Accordion>
-          )}
-        </>
```

Comment:
All this functionalities disappears! They are not added to the schema! this is breaking! please include it and do not remove functionalities

Reply:
ups, they somehow slipped away! Sure, we must include them.

Reply:
@danielamormocea

Reply:
@giuliaghisini @sneridagh We don't use this code in the current https://6.demo.plone.org/xv. I think that code is obsolete and meant to be cleaned up.

Reply:
> 

yess.. the code there is! 
You have to link an mp4 video. 

Here an example:
https://6.demo.plone.org/test-video

Reply:
okay for `.mp4`. Thank you!. 
-cc @danielamormocea